src/serve_seeds_api.ts

```diff
@@ -0,0 +1,37 @@
+import 'dotenv/config';
+import './types/env';
+import cors from 'cors';
+import express from 'express';
+
+import { persistSeed, validateSeed } from './types/seeds';
+import { authMiddleware } from './utils/seedsMiddleware';
+
+const apiVersionPrefix = '/v1';
+
+const app = express();
+app.use(express.json());
+app.use(cors({ origin: true }));
+app.use(authMiddleware);
+
+app.post(`${apiVersionPrefix}/seeds/`, async (req, res) => {
+  try {
+    validateSeed(req.body)
+  } catch (e: any) {
+    return res.status(422).send({ error: e.message });
+  }
+
+  try {
+    await persistSeed(req.body)
+    res.sendStatus(200);
+  } catch (e: any) {
+    res.status(500).send({ error: e.message });
+  }
+});
+
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
I'm not really familiar with supertest, but yeah agree - maybe we can find a workaround/config on it so that we don't need to add this custom test conditional